packages/destination-actions/src/destinations/personas-messaging-sendgrid/sendEmail/index.ts

```diff
@@ -146,8 +146,18 @@ const action: ActionDefinition<Settings, Payload> = {
       type: 'object',
       required: false
     },
+    send: {
+      label: 'Send Message',
+      description: 'Send Message can have true/false based on to send message or not',
```

Comment:
non-blocking suggestion
```suggestion
      label: 'Send Message?',
      description: 'Whether or not the message should actually get sent.',
```